readyplayerme/rpm-unity-sdk-core
# Release/v7.2.0

## [TICKETID](https://ready-player-me.atlassian.net/browse/TICKETID)

## Description

-   Preserve tag added to IAssetData for issues related to code stripping for android builds
-  response data handling updated for requests to reduce garbage allocation






## How to Test

Install sdk from this url 
`https://github.com/readyplayerme/rpm-unity-sdk-core.git#release/v7.2.0`

-   Test Wizard avatar creator sample on android with code stripping higher then minimum and ensure no errors
-  For response data update test all sample scenes in editor and on multiple platform builds to ensure no errors have been introduced for web requests. 



## Checklist
